subworkflows/local/frameshift_correction.nf

```diff
@@ -0,0 +1,74 @@
+include { PROKKA as PROKKA_BEFORE } from '../../modules/local/prokka'
+include { PROKKA as PROKKA_AFTER  } from '../../modules/local/prokka'
+include { DIAMOND_MAKEDB        } from '../../modules/nf-core/diamond/makedb/main'
+include { DIAMOND_BLASTP as DIAMOND_BLASTP_BEFORE } from '../../modules/nf-core/diamond/blastp/main'
+include { DIAMOND_BLASTP as DIAMOND_BLASTP_AFTER } from '../../modules/nf-core/diamond/blastp/main'
+include { IDEEL as IDEEL_BEFORE } from '../../modules/local/ideel'
+include { IDEEL as IDEEL_AFTER  } from '../../modules/local/ideel'
+include { PROOVFRAME_FIX        } from '../../modules/nf-core/proovframe/fix/main'
+include { PROOVFRAME_MAP        } from '../../modules/nf-core/proovframe/map/main'
+
+workflow FRAMESHIFT_CORRECTION {
+    take:
+    contigs                   // [ val(meta), path(contigs) ]
+
+    main:
+    ch_versions = Channel.empty()
+
+    PROKKA_BEFORE(
+        contigs
+    )
+    ch_versions = ch_versions.mix(PROKKA_BEFORE.out.versions)
+
+    DIAMOND_MAKEDB(
+        [[ id:'diamond_db' ], file(params.diamond_db, checkIfExists: true) ], [], [], []
+    )
+    ch_versions = ch_versions.mix(DIAMOND_MAKEDB.out.versions)
```

Comment:
Can't we accept an already built DB? That would save loads of compute/storage

Reply:
Yes good idea. Should we keep the option for users to create it, or totally remove it? 
I can also ask next week at EMG

Reply:
You already have the code there, so maybe just keep that. Overall having that across all the references would be good, in case we are missing an index we can just provide the genome. This comes at the expense of complexity for the pipeline, which I prefer to keep controlled. :thinking: